dashboard/state_aggregator.py

```diff
@@ -0,0 +1,130 @@
+import logging
+
+from dataclasses import fields
+
+import ray.dashboard.utils as dashboard_utils
+from ray.core.generated import gcs_service_pb2
+from ray.core.generated import gcs_service_pb2_grpc
+from ray.experimental.state.common import (
+    ActorState,
+    PlacementGroupState,
+    NodeState,
+    WorkerState,
+)
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_RPC_TIMEOUT = 30
+
+
+class GcsStateAggregator:
+    def __init__(self, gcs_channel):
+        self._gcs_actor_info_stub = gcs_service_pb2_grpc.ActorInfoGcsServiceStub(
```

Comment:
No preference here. I used stub because it is the name used within other parts of dashboard + this is actually called stub in API (e.g., `ActorInfoGcsServiceStub `). If you'd prefer client, I can change it, so lmk!